.golangci.yml

```diff
@@ -36,3 +36,19 @@ issues:
   max-issues-per-linter: 50
   max-same-issues: 10
   new: false
+  exclude-rules: 
+    # False positive httptest.NewRecorder
+    - path: 'gateway/webhook/webhook_test.go'
+      linters:
+        - bodyclose
+  
+    # False positive .Close behind if
+    - path: 'processor/transformer/transformer.go'
+      linters:
+        - bodyclose
+    
+    # False positive httptest.NewRecorder
+    - path: 'gateway/gateway_test.go'
+      linters:
+        - bodyclose
+
```

Comment:
bodyclose linter is very useful, but has a lot of false positives. I try to skip specific false positives without polluting the code.